aws-api/src/main/java/com/amplifyframework/api/aws/ApiAuthProviders.java

```diff
@@ -74,6 +77,28 @@ public CognitoUserPoolsAuthProvider getCognitoUserPoolsAuthProvider() {
         return this.cognitoUserPoolsAuthProvider;
     }
 
+    /**
+     * Returns a list of the auth providers currently configured.
+     * @param apiConfiguration A reference to the API configuration.
+     * @return a list of {@link AuthorizationType}
+     */
+    public List<AuthorizationType> getAvailableAuthorizationTypes(ApiConfiguration apiConfiguration) {
```

Comment:
If the order doesn't matter, `Set` is probably the way to go here.

Reply:
You're correct. The order here is not super relevant so I should be able to change to a set here.

Reply:
Done.